app/models/grda_warehouse/hud/client.rb

```diff
@@ -1145,14 +1149,15 @@ def disabling_condition?
     def domestic_violence?
       return pathways_domestic_violence if pathways_domestic_violence
 
-      dv_scope = source_health_and_dvs.where(DomesticViolenceVictim: 1)
+      dv_scope = source_health_and_dvs.select { |m| m.DomesticViolenceVictim == 1 }
```

Comment:
Comment about not using `where` to allow preloads here too? Maybe at the top of the section, and there are a bunch of places where you changed that?